packages/opentelemetry-sdk-trace-base/src/utility.ts

```diff
@@ -61,21 +70,31 @@ export function mergeConfig(userConfig: TracerConfig): TracerConfig & {
 export function reconfigureLimits(userConfig: TracerConfig): TracerConfig {
   const spanLimits = Object.assign({}, userConfig.spanLimits);
 
+  // parse environment in node or browser
+  const parsedEnvConfig =
+    typeof process !== 'undefined' ?
+      parseEnvironment(process.env as RAW_ENVIRONMENT) :
```

Comment:
Sorry for the churn. I'd suggest moving this environ source variable selection to `@opentelemetry/core` just like `getEnv`. This can help us reduce the pain to parse environs.

Reply:
Done